eng/Versions.DotNetScaffold.props

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <VersionPrefix>9.0.0</VersionPrefix>
-    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>10</PreReleaseVersionIteration>
+    <VersionPrefix>10.0.0</VersionPrefix>
```

Comment:
We should probably discuss with the team whether we want to tie vNext of dotnet-scaffold to v10 yet, or if an interim version is more appropriate for what we are targeting, before we ship a v10 preview to one of the feeds. Not the end of the world if we do ship it but doesn't hurt to have the conversation first I think.

Reply:
sure yes can switch this back for now, will check how the darc feeds are flowing.